client/src/components/chat/chatTab.js

```diff
@@ -72,21 +105,27 @@ export default class chatTab extends Component {
         this.socket.emit(`client:sendMessage`, { text: message, chatId:this.props.chatId})
     }
 
+    notifyTyping () {
+        this.socket.emit(`client:isTyping`)
+    }
+
     render() {
         return (
         <div className="mini-container">
-            {this.props.isSuperuser ?
-                <h2>
-                    {Strings.chat.chatWith} {this.state.chatClientName}
-                </h2>
-                :
-                <h2>
-                    {Strings.chat.chatWithTech}
-                </h2>
-
-            }
+            <h2>
+                {
+                    this.props.isSuperuser
+                    ? `${Strings.chat.chatWith} ${this.state.chatClientName}`
+                    : Strings.chat.chatWithTech
+                }
+            </h2>
 
             <div className="chat-area" >
+                {this.state.showIsTyping !== '' && (<div><p className="system-message">{this.state.showIsTyping}</p></div>)}
```

Comment:
say can you remind me what do we get from adding a <p>, that can't go into the div+class?